File: 백준/Bronze/2903. 중앙 이동 알고리즘/중앙 이동 알고리즘.java
import java.util.Scanner;

public class Main {
    public static int edgeDot(int n) {
        if(n == 1) {
            return 3;
        } else {
            return edgeDot(n - 1) + (edgeDot(n - 1) - 1);
        }
    }
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        System.out.println(edgeDot(n) * edgeDot(n));
    }
}

File: 백준/Bronze/2903. 중앙 이동 알고리즘/중앙 이동 알고리즘.cc
#include <iostream>

using namespace std;

int main() {
	
	int n;
	cin >> n;

	int edgeDot = 2;
	
	for (int i = 0; i < n; i++)
	{
		edgeDot = edgeDot * 2 - 1;
	}

	cout << edgeDot * edgeDot;

	return 0;
}